onnxruntime/python/tools/transformers/convert_generation.py

```diff
@@ -1304,6 +1304,24 @@ def update_decoder_subgraph_share_buffer_and_use_decoder_masked_mha(subg: GraphP
     return True
 
 
+def gpt2_switch_to_skip_layernorm_strict_mode(subg: GraphProto):
```

Comment:
Need not update graph. Instead need set provider option during creating session.